x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/pages/coverage_overview/technique_panel_popover.tsx

```diff
@@ -0,0 +1,183 @@
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const closePopover = useCallback(() => setIsPopoverOpen(false), []);
+  const coveredSubtechniques = useMemo(() => getCoveredSubtechniques(technique), [technique]);
+
+  const TechniquePanel = useMemo(
+    () => (
+      <CoverageOverviewMitreTechniquePanel
+        setIsPopoverOpen={setIsPopoverOpen}
+        isPopoverOpen={isPopoverOpen}
+        technique={technique}
+        isExpanded={isExpanded}
+        coveredSubtechniques={coveredSubtechniques}
+      />
+    ),
+    [technique, isPopoverOpen, isExpanded, coveredSubtechniques]
+  );
+
+  const CoveredSubtechniquesLabel = useMemo(
+    () => (
+      <EuiText color="success" size="s">
+        <h4>
+          {i18n.COVERED_MITRE_SUBTECHNIQUES(coveredSubtechniques, technique.subtechniques.length)}
+        </h4>
+      </EuiText>
+    ),
+    [coveredSubtechniques, technique.subtechniques.length]
+  );
+
+  const enabledRuleListItems: EuiListGroupItemProps[] = useMemo(
+    () =>
+      technique.enabledRules.map((rule) => ({
+        label: rule.name,
+        color: 'primary',
+        showToolTip: true,
+        onClick: () => {},
+      })),
+    [technique.enabledRules]
+  );
+
+  const disabledRuleListItems: EuiListGroupItemProps[] = useMemo(
+    () =>
+      technique.disabledRules.map((rule) => ({
+        label: rule.name,
+        color: 'primary',
+        showToolTip: true,
+        onClick: () => {},
+      })),
+    [technique.disabledRules]
+  );
+
+  const EnabledRulesAccordionButton = useMemo(
+    () => (
+      <CoverageOverviewRuleListHeader
+        listTitle={i18n.ENABLED_RULES_LIST_LABEL}
+        listLength={technique.enabledRules.length}
+      />
+    ),
+    [technique.enabledRules.length]
+  );
+
+  const DisabledRulesAccordionButton = useMemo(
+    () => (
+      <CoverageOverviewRuleListHeader
+        listTitle={i18n.DISABLED_RULES_LIST_LABEL}
+        listLength={technique.disabledRules.length}
+      />
+    ),
+    [technique.disabledRules.length]
+  );
+
+  return (
+    <EuiPopover
+      button={TechniquePanel}
+      isOpen={isPopoverOpen}
+      closePopover={closePopover}
+      anchorPosition="rightCenter"
+      data-test-subj="coverageOverviewPopover"
+      ownFocus={false}
+    >
+      <EuiPopoverTitle>
+        <EuiFlexGroup gutterSize="xs" alignItems="flexStart" direction="column">
+          <EuiFlexItem>
+            <EuiButtonEmpty
+              flush="left"
+              iconType="popout"
+              iconSide="right"
+              href={technique.reference}
+              target="_blank"
+            >
+              <EuiText>
+                <h3>{technique.name}</h3>
+              </EuiText>
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+          <EuiFlexItem>{CoveredSubtechniquesLabel}</EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiPopoverTitle>
+      <div className="eui-yScrollWithShadows" css={{ maxHeight: '700px', padding: '5px 0px' }}>
```

Comment:
I set it to an `em` constant instead. The padding was chosen because that eui classname that adds the fade in for scrolling isn't properly aligned with the popover section so it spills over into the content making it look scrollable when it's not. The padding basically realigns it